src/main/webapp/app/complaints/complaint-interactions.component.ts

```diff
@@ -124,20 +124,37 @@ export class ComplaintInteractionsComponent implements OnInit {
      */
     get isTimeOfFeedbackRequestValid(): boolean {
         if (!this.isExamMode && this.result && this.result.completionDate) {
-            return this.isComplaintWithinTimeBoundaries();
+            return this.isMoreFeedbackRequestWithinTimeBoundaries();
+        }
+        return false;
+    }
+
+    /**
+     * Checks if a more feedback request can be filed
+     */
+    isMoreFeedbackRequestWithinTimeBoundaries(): boolean {
+        if (this.course?.requestMoreFeedbackEnabled && this.course?.maxRequestMoreFeedbackTimeDays !== undefined) {
+            const resultCompletionDate = dayjs(this.result.completionDate!);
+            if (!this.exercise.assessmentDueDate || resultCompletionDate.isAfter(this.exercise.assessmentDueDate)) {
+                return resultCompletionDate.isAfter(dayjs().subtract(this.course?.maxRequestMoreFeedbackTimeDays, 'day'));
+            }
+            return dayjs(this.exercise.assessmentDueDate).isAfter(dayjs().subtract(this.course?.maxRequestMoreFeedbackTimeDays, 'day'));
```

Comment:
This looks like the same as the one below => extract to a method `isWithinTimeBoundaries()` that takes a parameter for the subtraction.

Reply:
this is a style issue that can be addressed later on. I also think that adding a new method here is not really required, as it's only two one-liners anyways.

Reply:
I'd disagree because it is not two one-liners, it's two five-liners. I marked all lines that should be extracted.

However, it can indeed be addressed later on. Did you already create an issue for that like for the translations?